### DISCUSSION TITLE
Snooze alarm available?

### DISCUSSION THREAD

**Discussion by mhnisportal** [creepymonster/GlucoseDirect#561] [Q&A]
Hello and thank you for the work on this awesome app!

I was wondering if it is possible to snooze an alarm. I ask because, this is what I want:
* I want alarms (well, only low, really) to override mute/dnd, because I want to be protected from lows while I sleep.
* I want to be able to snooze alarms, as I am often in meetings and want to wait 5-10mins to deal with a high/low.

Is the above possible to do in the current app?

Thank you.

**Comment by dlongman** [comment]
You can already do this by clicking on the alarm icon under the glucose reading on the right hand side. When you do this the time will change to show the snoozed time. The more times you press it the longer it is snoozed for.
